<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QEventWidget</class>
 <widget class="QWidget" name="QEventWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Precondition</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkSwitchA">
          <property name="text">
           <string>Switch</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="SwitchRpgComboBox" name="comboSwitchA">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="SwitchRpgComboBox" name="comboSwitchB">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkSwitchB">
          <property name="text">
           <string>Switch</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkVar">
          <property name="text">
           <string>Variable</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="VariableRpgComboBox" name="comboVariable">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QComboBox" name="comboVarOperation">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>4</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinVarValue">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>3</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="checkItem">
          <property name="text">
           <string>Item</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ItemRpgComboBox" name="comboItem">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="checkHero">
          <property name="text">
           <string>Hero</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ActorRpgComboBox" name="comboHero">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkTimerA">
          <property name="text">
           <string>Timer 1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinTimerAMin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> Minutes</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="spinTimerASec">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> seconds or less</string>
          </property>
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkTimerB">
          <property name="text">
           <string>Timer 2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinTimerBMin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> Minutes</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="spinTimerBSec">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> seconds or less</string>
          </property>
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3" rowspan="5">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Event Commands</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="EventCommandsWidget" name="commands" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Event Graphic</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QGraphicsView" name="graphicsSprite">
        <property name="minimumSize">
         <size>
          <width>52</width>
          <height>68</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>52</width>
          <height>68</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QCheckBox" name="checkTransparent">
          <property name="text">
           <string>Trans</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushSetSprite">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBox_7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Trigger Condition</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboCondition"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QGroupBox" name="groupBox_8">
     <property name="title">
      <string>Event Layer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboLayer"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkOverlapForbidden">
        <property name="text">
         <string>Forbid overlap</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Movement Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboMoveType"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Frequency</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboMoveFrequency">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushMovePattern">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Define Pattern</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox_9">
     <property name="title">
      <string>Animation type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboAnimationType"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox_10">
     <property name="title">
      <string>Move Speed</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboMoveSpeed"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EventCommandsWidget</class>
   <extends>QWidget</extends>
   <header>ui/event/event_commands_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SwitchRpgComboBox</class>
   <extends>QComboBox</extends>
   <header>ui/common/rpg_combobox.h</header>
  </customwidget>
  <customwidget>
   <class>VariableRpgComboBox</class>
   <extends>QComboBox</extends>
   <header>ui/common/rpg_combobox.h</header>
  </customwidget>
  <customwidget>
   <class>ActorRpgComboBox</class>
   <extends>QComboBox</extends>
   <header>ui/common/rpg_combobox.h</header>
  </customwidget>
  <customwidget>
   <class>ItemRpgComboBox</class>
   <extends>QComboBox</extends>
   <header>ui/common/rpg_combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkSwitchA</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboSwitchA</receiver>
   <slot>setEnabled(bool)</slot>
  </connection>
  <connection>
   <sender>checkSwitchB</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboSwitchB</receiver>
   <slot>setEnabled(bool)</slot>
  </connection>
  <connection>
   <sender>checkVar</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboVariable</receiver>
   <slot>setEnabled(bool)</slot>
  </connection>
  <connection>
   <sender>checkVar</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinVarValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkItem</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboItem</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkHero</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboHero</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkTimerA</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinTimerAMin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkTimerA</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinTimerASec</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkTimerB</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinTimerBMin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkTimerB</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinTimerBSec</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
